pkgs/halo2-eth-membership-worker/src/worker.ts

```diff
@@ -19,21 +21,20 @@ export const Halo2EthMembershipWorker: IHalo2EthMembershipWorker = {
     const response = await fetch(wasmModuleUrl)
     const bufferSource = await response.arrayBuffer()
 
-    await halo2EthMembershipWasm.initSync(bufferSource)
-    await halo2EthMembershipWasm.initPanicHook()
+    halo2EthMembershipWasm.initSync(bufferSource)
+    halo2EthMembershipWasm.initPanicHook()
```

Comment:
like we had tested in another PR, the `await` have no effect here and aren't necessary.